security/org-account-context/main.tf

```diff
@@ -276,3 +276,368 @@ module "github_oidc_provider" {
+    "EC2"                    = true
+    "EFS"                    = true
+    "FSx"                    = false
+    "Neptune"                = false
+    "RDS"                    = true
+    "Redshift"               = true
+    "S3"                     = true
+    "SAP HANA on Amazon EC2" = false
+    "Storage Gateway"        = false
+    "Timestream"             = true
+    "VirtualMachine"         = false
+  }
+  resource_type_management_preference = {
+    "DynamoDB" = true
+    "EFS"      = true
+  }
+  vault_name     = "dfds-vault"
+  deploy_kms_key = true
+  kms_key_admins = [module.org_account.org_role_arn]
+
+  backup_plans = [
+    {
+      plan_name = "30-days-retention-continuous-backup"
+      rules = [
+        {
+          name                     = "30-days-retention-continuous-backup"
+          schedule                 = "cron(0 1 * * ? *)"
+          enable_continuous_backup = true
+          lifecycle = {
+            delete_after = 30 # Max 35 days for continuous backup
+          }
+        }
+      ]
+      selections = [
+        {
+          name      = "select-rds-cont-backup"
+          resources = ["arn:aws:rds:*:*:db:*"]
+          conditions = {
+            string_equals = [
+              {
+                key   = "aws:ResourceTag/dfds.data.backup"
+                value = "true"
+              }
+            ]
+          }
+        }
+      ]
+    },
+    {
+      plan_name = "30-days-retention"
+      rules = [
+        {
+          name                     = "daily-30-days-retention"
+          schedule                 = "cron(0 1 * * ? *)"
+          enable_continuous_backup = false
+          lifecycle = {
+            delete_after = 30
+          }
+        }
+      ]
+      selections = [
+        {
+          name = "select-prod-30-days"
+          resources = [
+            "arn:aws:dynamodb:*:*:table/*",
+            "arn:aws:rds:*:*:db:*"
+          ]
+          conditions = {
+            string_equals = [
+              {
+                key   = "aws:ResourceTag/dfds.env"
+                value = "prod"
+              },
+              {
+                key   = "aws:ResourceTag/dfds.data.backup"
+                value = "true"
+              },
+              {
+                key   = "aws:ResourceTag/dfds.data.backup_retention"
+                value = "30days"
+              }
+            ]
+          }
+        },
+        {
+          name = "select-staging-30-days"
+          resources = [
+            "arn:aws:dynamodb:*:*:table/*",
+            "arn:aws:rds:*:*:db:*"
+          ]
+          conditions = {
+            string_equals = [
+              {
+                key   = "aws:ResourceTag/dfds.env"
+                value = "staging"
+              },
+              {
+                key   = "aws:ResourceTag/dfds.data.backup"
+                value = "true"
+              },
+              {
+                key   = "aws:ResourceTag/dfds.data.backup_retention"
```

Comment:
The ResourceTag should be a variable since infrastructure-module is an open source project, and I guess any external user would prefer to specify their own ResourceTag name,